src/clamp.js

```diff
@@ -0,0 +1,27 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Clamps a number within a range.
```

Comment:
In light of #1578 we should probably devote one sentence to the common case and one sentence to the abstraction. What do you think, @CrossEye?